.travis.yml

```diff
@@ -24,6 +24,18 @@ cache:
   - "$HOME/.ivy2"
   - "$HOME/.sbt"
 
+services:
+   - docker
+
+before_install:
+  - cat /etc/hosts # optionally check the content *before*
+  - sudo hostname "$(hostname | cut -c1-63)"
+  - sed -e "s/^\\(127\\.0\\.0\\.1.*\\)/\\1 $(hostname | cut -c1-63)/" /etc/hosts | sudo tee /etc/hosts
+  - cat /etc/hosts # optionally check the content *after*
+
+before_script:
+  - docker run -d --restart=always --net=host -m 1G --memory-swap -1 --env="MAX_HEAP_SIZE=500M" --env="HEAP_NEWSIZE=100M" --env="CASSANDRA_LISTEN_ADDRESS=127.0.0.1" --name dc-cassandra cassandra:latest
```

Comment:
This should be a script that is referred to in error message for the tests, it can be handled in the same matter as slick tests. Otherwise you have to know to run this command to run the local tests.